CodeEdit/NavigatorSidebar/ProjectNavigator/OutlineView/OutlineViewController.swift

```diff
@@ -35,6 +35,9 @@ final class OutlineViewController: NSViewController {
     var workspace: WorkspaceDocument?
 
     var iconColor: AppPreferences.FileIconStyle = .color
+    var fileExtension: AppPreferences.FileExtensions = .showAll
```

Comment:
Just a suggestion: maybe rename it to something more descriptive since the property technically isn't a file extension.

Maybe something like `fileExtensionVisibility`.

Reply:
Renamed In [8a9d07c](https://github.com/CodeEditApp/CodeEdit/commit/8a9d07c961902a31ec856161bd4f44512a483d8e)